### PR TITLE
PEM: Adding db server matrix as per PEM-4741

### PR DESCRIPTION
**Pull request by nidhibhammar**

### CODE REVIEW

File: product_docs/docs/pem/8/supported_platforms.mdx [hunk 1]
```diff
@@ -8,4 +8,14 @@ For information about the platforms and versions supported by PEM, see [Platform
 !!! Note
     Postgres Enterprise Manager 8.3 and later is supported on SLES.
 
-## Database compatibility
\ No newline at end of file
+## Database compatibility
+
+The following table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD) for PEM 8.x.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/supported_platforms.mdx:13]: Change "The following table" to "This table"

File: product_docs/docs/pem/8/supported_platforms.mdx [hunk 2]
```diff
@@ -8,4 +8,14 @@ For information about the platforms and versions supported by PEM, see [Platform
...
+
+|           |**PEM 8.x**                   |                    |
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/supported_platforms.mdx:15]: The intro sentence mentions PEM 8.x and also this doc is for PEM v8 so I think this header is not required here

File: product_docs/docs/pem/8/supported_platforms.mdx [hunk 3]
```diff
@@ -8,4 +8,14 @@ For information about the platforms and versions supported by PEM, see [Platform
...
+|:----------|:-----------------------------|:-------------------|
+|           |**As a monitored instance**   |**As a backend**    |
+|**PG**     |11, 12, 13, 14                |11, 12, 13, 14      |
+|**PGE**    |11, 12, 13, 14                |11, 12, 13, 14      |
+|**EPAS**   |11, 12, 13, 14                |11, 12, 13, 14      |
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/supported_platforms.mdx:20]: I think we can take EPAS in first row and then in 2nd and 3rd row PG and PGE respectively

File: product_docs/docs/pem/9/supported_platforms.mdx [hunk 4]
```diff
@@ -9,3 +9,13 @@ For information about the platforms and versions supported by PEM, see [Platform
     Postgres Enterprise Manager 8.3 and later is supported on SLES.
 
 ## Database compatibility
+
+The following table provides information about the PEM versions and their supported corresponding versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/supported_platforms.mdx:13]: Keep the same intro sentence as PEM 8.x

File: product_docs/docs/pem/9/supported_platforms.mdx [hunk 5]
```diff
@@ -9,3 +9,13 @@ For information about the platforms and versions supported by PEM, see [Platform
...
+
+|           |**PEM 8.x**                   |                    |**PEM 9.x**                    |                         |
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/supported_platforms.mdx:15]: I think we don't need PEM 8.x here as it is already added in PEM 8 docs. Same comments as above for header and reordering the database server names.

File: product_docs/docs/pem/8/supported_platforms.mdx (hunk 1 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/8/supported_platforms.mdx]: Done

File: product_docs/docs/pem/8/supported_platforms.mdx (hunk 2 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/8/supported_platforms.mdx]: Done

File: product_docs/docs/pem/8/supported_platforms.mdx (hunk 3 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/8/supported_platforms.mdx]: Done

File: product_docs/docs/pem/9/supported_platforms.mdx (hunk 4 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/supported_platforms.mdx]: Done

File: product_docs/docs/pem/9/supported_platforms.mdx (hunk 5 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/supported_platforms.mdx]: Done

File: product_docs/docs/pem/8/supported_platforms.mdx [hunk 6]
```diff
@@ -10,12 +10,12 @@ For information about the platforms and versions supported by PEM, see [Platform
 
 ## Database compatibility
 
-The following table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD) for PEM 8.x.
+This table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/supported_platforms.mdx:13]: Add " for PEM 8.x" at the last of intro sentence  and remove from the table header

File: product_docs/docs/pem/9/supported_platforms.mdx [hunk 7]
```diff
@@ -10,12 +10,12 @@ For information about the platforms and versions supported by PEM, see [Platform
 
 ## Database compatibility
 
-The following table provides information about the PEM versions and their supported corresponding versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
+This table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/supported_platforms.mdx:13]: Add " for PEM 9.x" at the last of intro sentence and remove from the table header

File: product_docs/docs/pem/8/supported_platforms.mdx (hunk 6 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/8/supported_platforms.mdx]: Done

File: product_docs/docs/pem/9/supported_platforms.mdx (hunk 7 above)
Comment (arup-roy2023) [reply product_docs/docs/pem/9/supported_platforms.mdx]: Done

File: product_docs/docs/pem/8/supported_platforms.mdx [hunk 8]
```diff
@@ -8,6 +8,13 @@ For information about the platforms and versions supported by PEM, see [Platform
 !!! Note
     Postgres Enterprise Manager 8.3 and later is supported on SLES.
 
-## Supported locales
+## Database compatibility
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/supported_platforms.mdx:11]: I think this should be Postgres compatibility and that we should spell out the product names in the table. I think there is room for that.

File: product_docs/docs/pem/9/supported_database_versions.mdx [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+This table provides information about the supported versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD) for PEM 9.x. 
+
+|           |**Monitored Instance**      |**Backend Instance**    |
+|:----------|:---------------------------|:-----------------------|
+|**EPAS**   |11, 12, 13, 14, 15          |11, 12, 13, 14, 15      |
+|**PG**     |11, 12, 13, 14, 15          |11, 12, 13, 14, 15      |
+|**PGE**    |11, 12, 13, 14, 15          |11, 12, 13, 14, 15      |
+|**PGD**    |3, 4, 5[^1]                 |                        |
+
+[^1]: From PEM version 9.1 and later, EDB Postgres Distributed (PGD) 5 is supported.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/supported_database_versions.mdx:14]:
```suggestion
[^1]: PEM version 9.1 and later supports EDB Postgres Distributed (PGD) 5.
```

File: product_docs/docs/pem/8/index.mdx [hunk 10]
```diff
@@ -58,4 +58,16 @@ redirects:
 
 Welcome to Postgres Enterprise Manager (PEM). PEM consists of components that provide the management and analytical functionality for your EDB Postgres Advanced Server or PostgreSQL database. PEM is based on the Open Source pgAdmin 4 project.
 
-PEM is a comprehensive database design and management system. PEM is designed to meet the needs of both novice and experienced Postgres users alike, providing a powerful graphical interface that simplifies the creation, maintenance, and use of database objects.
+PEM is a comprehensive database design and management system. PEM is designed to meet the needs of both novice and experienced Postgres users alike, providing a powerful graphical interface that simplifies the creation, maintenance, use of database objects and monitoring multiple postgres servers through a single graphical interface.
+
+## Postgres compatibility
+
+The table lists the compatibility matrix information for PEM 8.x.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/index.mdx:65]:
```suggestion
The table provides information about the supported versions of Postgres for PEM 8.x.
```

**Comment by drothery-edb** [comment]
To not complicate the feedback we might get when asking for input on where to put the information, let's use the same format for the sections for each example. I like the stem sentence I suggested in the v8 example and spelling out the product names in the first column. We should add the footnote to the v8 example.

**Comment by nidhibhammar** [comment]
@sonotley, We have added the matrix for PEM 8 on the landing page, and for PEM 9 in a separate topic, "Postgres compatibility". I would like to know what is your preference.

**Comment by sonotley** [comment]
> I would like to know what is your preference.

I don't have a strong preference. The way you have done it for v8 is consistent with PGD so unless you plan to change PGD I would stick with that. You could also use the words 'compatibility matrix' somewhere so the same search term finds the equivalent info.

**Comment by arup-roy2023** [comment]
Based on Simon's comments and suggestion from Nidhi, I have moved the "Postgres compatibility" with the matrix table from the 'Supported Database Version' to the index file, similar to v8. This is to maintain consistency. Please review. As there were no other content except the matrix table in the 'Supported Database Version' file, I have removed it.

File: product_docs/docs/pem/8/index.mdx [hunk 11]
```diff
@@ -58,4 +58,16 @@ redirects:
 
 Welcome to Postgres Enterprise Manager (PEM). PEM consists of components that provide the management and analytical functionality for your EDB Postgres Advanced Server or PostgreSQL database. PEM is based on the Open Source pgAdmin 4 project.
 
-PEM is a comprehensive database design and management system. PEM is designed to meet the needs of both novice and experienced Postgres users alike, providing a powerful graphical interface that simplifies the creation, maintenance, and use of database objects.
+PEM is a comprehensive database design and management system. PEM is designed to meet the needs of both novice and experienced Postgres users alike, providing a powerful graphical interface that simplifies the creation, maintenance, use of database objects and monitoring multiple postgres servers through a single graphical interface.
+
+## Postgres compatibility
+
+The table provides information about the supported versions of Postgres for PEM 8.x.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/index.mdx:65]:
```suggestion
Supported versions of Postgres for PEM 8.x:
```

File: product_docs/docs/pem/9/index.mdx [hunk 12]
```diff
@@ -58,3 +59,15 @@ redirects:
 Welcome to Postgres Enterprise Manager (PEM). PEM consists of components that provide the management and analytical functionality for your EDB Postgres Advanced Server or PostgreSQL database. PEM is based on the Open Source pgAdmin 4 project.
 
 PEM is a comprehensive database design and management system. PEM is designed to meet the needs of both novice and experienced Postgres users alike, providing a powerful graphical interface that simplifies the creation, maintenance, and use of database objects.
+
+## Postgres compatibility
+
+The table provides information about the supported versions of Postgres for PEM 9.x.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/index.mdx:65]:
```suggestion
Supported versions of Postgres for PEM 9.x:
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**